src/server/scripts/EasternKingdoms/Scholomance/boss_illucia_barov.cpp

```diff
@@ -30,8 +30,8 @@ enum Spells
 {
     SPELL_CURSEOFAGONY          = 34794,
     SPELL_SHADOWSHOCK           = 34799,
-    SPELL_SILENCE               = 34803,
-    SPELL_FEAR                  = 34803
+    SPELL_SILENCE               = 15487,
```

Comment:
this needs to be confirmed with sniffes, even curse of agony and shadow shock have wrong spellids